app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -214,16 +215,31 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
         }
 
         _webView.setBackgroundColor(Color.TRANSPARENT);
-        final Bundle args = getArguments();
-        setViewModeVisibility(args.getBoolean(START_PREVIEW, _appSettings.getDocumentPreviewState(_document.getPath())));
+    }
 
-        _loadModTime = 0; // force next load
-        loadDocument();
+    // Runs after all state has been restored (i.e. text is restored etc)
+    @Override
+    public void onViewStateRestored(final Bundle savedInstanceState) {
```

Comment:
Moved several things to `onViewStateRestored` so they happen correctly later in the lifecycle